DeepMenu/Pod/MuPodModel.swift

```diff
@@ -3,23 +3,38 @@
 import SwiftUI
 
 /// shared between 1 or more MuPod, stored
-class MuPodModel: Identifiable {
+class MuPodModel: Identifiable, Equatable {
     let id = MuIdentity.getId() //TODO: use a persistent storage ID after debugging
     
     var name: String // pilot's name may be changed hubd on corner placement
     let title: String
     let type: MuBorderType
-    var subModels: [MuPodModel]?
+    var subModels = [MuPodModel]()
```

Comment:
Making this non-optional cleaned up some conditionals elsewhere. I'm not sure if it has downsides though?